spec/requests/apartments_spec.rb

```diff
@@ -20,18 +21,53 @@
           pets: 'No',
           image: 'https://photos.zillowstatic.com/fp/7f7209dfbb519e5aa13446bbbd60c43d-cc_ft_576.webp',
           user_id: user1.id
+      }
     }
-  }
       post '/apartments', params: apartment_params
 
       apartment = JSON.parse(response.body)
       expect(response).to have_http_status(200)
       expect(apartment['street']).to eq '1329 Carroll Avenue'
     end
+  
+
+    it "doesn't create an apartment without valid attributes" do
```

Comment:
Usually, you would write a seperate it test for each individual column because you want to also verify the HTTP status if a single column missing information will throw that error.